tox_travis.py

```diff
@@ -67,10 +68,33 @@ def guess_python_env():
     return 'py{major}{minor}'.format(major=major, minor=minor)
 
 
+def get_default_envlist(version):
+    """Parse a default tox env based on the version.
+
+    The version comes from the ``TRAVIS_PYTHON_VERSION`` environment
+    variable. If that isn't set, then use sys.version_info to come
+    up with a reasonable default.
+    """
+    # If version isn't set, then guess from sys.version_info
+    if not version:
+        return guess_python_env()
```

Comment:
It should fallback to guessing if all else fails, like an exception occurs somewhere